src/main/transports/electron-offline-net.ts

```diff
@@ -0,0 +1,58 @@
+import { Response, Status, TransportOptions } from '@sentry/types';
+import { logger } from '@sentry/utils';
+import { net } from 'electron';
+import { join } from 'path';
+
+import { sentryCachePath } from '../fs';
+import { ElectronNetTransport, SentryElectronRequest } from './electron-net';
+import { PersistedRequestQueue } from './queue';
+
+/** Returns true is there's a chance we're online */
+function maybeOnline(): boolean {
+  return !('online' in net) || net.online === true;
```

Comment:
Should we also check `navigator.onLine`?

Reply:
We are in nodejs context here

Reply:
Ah yeah woooooops lol